### PR TITLE
333 resize notebook

### LINKED ISSUE #333: Resize Notebook Port
We have a great notebook for the resize transformation, but it is for the old library!
https://github.com/opendp/smartnoise-samples/blob/master/analysis/unknown_dataset_size.ipynb

There is a partial port here that just needs to be extended:
https://github.com/opendp/opendp/blob/main/python/example/unknown_dataset_size.ipynb

### PR DESCRIPTION
**Pull request by ecowan**
Closes #333

### CODE REVIEW

**Review by Shoeboxam — CHANGES_REQUESTED** [review]
I made a pass through:
- made the old section of the notebook match with the new section
- switched the old section to use means instead of sums (fp is more stable now)
- pass Vec<f64> in, instead of parsing from a csv string. Simplifies the chains
- add bounds to the binary search calls to make things run quickly
- adjusted the epsilon value of the count estimate down, to fix the graphs
- rewrote most of the latter sections to simplify the implementation

We still need some more exposition around the boxes in the latter portion.

File: python/example/unknown_dataset_size.ipynb [hunk 1]
```diff
@@ -235,6 +406,67 @@
     "and the mean of the dataset."
    ]
   },
+  {
```
[the remark this answers is not in the document]
Comment (ecowan) [reply python/example/unknown_dataset_size.ipynb:352]: Agreed, I'm rewording

File: python/example/unknown_dataset_size.ipynb [hunk 2]
```diff
@@ -227,6 +232,172 @@
     "print(\"DP sum: {0}\".format(dp_sum_release))"
    ]
   },
+  {
```
[the remark this answers is not in the document]
Comment (ecowan) [reply python/example/unknown_dataset_size.ipynb:211]: I'm not sure which line in particular this is referring to?

[the remark this answers is not in the document]
Comment (Shoeboxam) [reply python/example/unknown_dataset_size.ipynb:211]: Cells 43-46 have no explanation/interpretation/accompanying text.

**Comment by Shoeboxam** [comment]
Force-pushed to rid history of the accuracy notebook, to fix the merge conflict.

**Review by Shoeboxam — APPROVED** [review]

**Pull request merged**